tests/examples/opinions/united_states/md_example.compare.json

```diff
@@ -7,9 +7,11 @@
     "blocked_statuses": false,
     "date_filed_is_approximate": false,
     "docket_numbers": "33ag/15",
-    "judges": "Order",
+    "judges": "",
     "citations": "",
-    "case_name_shorts": "Galbraith"
+    "case_name_shorts": "Galbraith",
+    "per_curiam": false,
+    "other_dates": ""
```

Comment:
Can you update the test file so that other_dates is used at least once?  I think its not captured in any of the 115 test examples

Reply:
It is used at least once, check the mdctspeapp_example_2.compare.json

https://github.com/grossir/juriscraper/blob/d2eb32f0e07e4a8bd44b6c695742ac59a80d4630/tests/examples/opinions/united_states/mdctspecapp_example_2.compare.json#L154